packages/big-bang-core/values.minimal.yaml

```diff
@@ -0,0 +1,46 @@
+domain: bigbang.dev
```

Comment:
Might be missing something but what's the proposed value of this minimal set? It looks like everything is disabled, is that accurate?